Makefile.common-jst

```diff
@@ -295,6 +295,14 @@ test: install_for_test
                fi \
              fi)
 
+promote-failed:
```

Comment:
I would expect this to need the `install_for_test` dependency. But that's just cargo-culted from something @stedolan did, so I might be wrong.

Reply:
(oops I have already merged it)

Yes I would want that as dependency as well; but `install_for_test` always create an empty `_runtest` dir which means `_runtest/_log` won't exist.

Reply:
Right -- this sort of "implicitly" depends on `test`, but the idea is it's run afterwards with human intervention.  `install_for_test` sets things up afresh and blows too much away.